en/guide/error-handling.md

```diff
@@ -62,7 +62,7 @@ Express regards the current request as being an error and will skip any
 remaining non-error handling routing and middleware functions.
 
 If the callback in a sequence provides no data, only errors, you can simplify
-this code as follows:
+write code as follows:
```

Comment:
Thank you, but unfortunately the phrase "you can simplify write code as follows" is not proper English. Is there another way you you would like to modify this?

Reply:
Oh sorry, I read that word simplify as simply, thats my eye error which I realised now. I would modify like this now -> 
'If the callback in a sequence provides no data, only errors, you can simplify the code as follows:' However original line also makes sense. 
Again sorry about my eye error.